service/resource_router_test.go

```diff
@@ -117,6 +117,7 @@ func Test_Service_NewResourceRouter(t *testing.T) {
 	versionedResources := make(map[string][]framework.Resource)
 	versionedResources["0.1.0"] = legacyResources
 	versionedResources["0.2.0"] = allResources
+	versionedResources["1.0.0"] = allResources
```

Comment:
is this still right?

Reply:
ping @rossf7 .. sorry Im not looking super critically. just seeing `allResources` next to `1.0.0` when we said it should use `legacyResources`. so just double checking

Reply:
yeah I guess test should be aligned

Reply:
Yes test should be aligned. Sorry I missed that and thanks for fixing Fer.

Reply:
I noticed this as well. IMO this is a sign for a bad test design. We test what we compute in the test setup. That makes not that much sense. We should rather test what we use in production kind of, which means that the resources we compute should be configured with stubs and tested here. Lets try to improve this in the future.